versions.json

```
@@ -1 +1,4 @@
-["v3.29.4", "v4.10.2"]
+[
+  "v2",
+  "v1"
+]
```

Right now this versioning adds noise to the repo - I would just leave the empty array without any document placeholder.

It will be fixed in #24 #25 etc..

```suggestion
[]
```

---

alright, this is noted